### ISSUE TITLE
Please make contact information fields, how to contact the user, email address, telegrams, various social networks https://www.openstreetmap.org

### ISSUE THREAD

**Issue by Sowa1980** [openstreetmap/openstreetmap-website#2245]
Please make contact information fields, how to contact the user, email address, telegrams, various social networks https://www.openstreetmap.org

Сделайте пожалуйста, поля для контактной информации, как связаться с пользователем, электронный адрес почты, телеграмм, различные соц сети https://www.openstreetmap.org

**Comment by tomhughes** [comment]
There are already tickets requesting this.

**Comment by tomhughes** [comment]
Well I thought there was anyway though I can't find them - they might be so old they're in trac rather than here.

The closest is #1274 but that's really about the social signin, and we don't expose those identities anyway.

We would need to be extremely careful (if only for GDPR compliance) that people were aware that the information was going to be public.

**Comment by simonpoole** [comment]
While conceivably allowing the users to add this kind of information as part of a more general way of allowing the user to be contacted, aka not just e-mail would be a good thing, we wouldn't be able to make it _available_  to other users without introducing some kind of opt in mechanism (which in itself would have all kind of complications). 

See see https://github.com/openstreetmap/openstreetmap-website/issues/2102 https://github.com/openstreetmap/openstreetmap-website/pull/1431 https://github.com/openstreetmap/openstreetmap-website/issues/908 for related discussion.

**Comment by mmd-osm** [comment]
Social networks is already there:

http://neis-one.org/2016/05/verified-osm-contributors/
https://www.openstreetmap.org/user/malenki/diary/38381

**Comment by Nakaner** [comment]
Doesn't adding fields for contact information (especially social networks) weaken the personal message feature of openstreetmap.org? At the moment, we expect any mapper to be reachable via personal messages and changeset comments, don't we?

**Comment by simonpoole** [comment]
@Nakaner at least previous discussion always resolved around -notifying- users or new communications via different means than just e-mail and not providing these as alternatives to the existing facilities.

**Comment by Sowa1980** [comment]
And why not let the users themselves decide that this information is available or not available at his request, I personally do not mind it.

А почему не дать пользователям сами решать что бы данная информация была доступна или не доступна по его желанию, я вот лично не против этого.

**Comment by nertc** [comment]
I agree with the issue, current User Profile page provides functionalities for social interactions like Friendship, Messaging, Profile Description and etc. But to strengthen up community beyond the website and ease communication, it would be great if Social Profile Links could be added to the profile page. For example, GitHub has such functionality on the profile page. Of course, it should be optional, and user should have ability to add, delete or modify Social Profile Links.

One way, I think, to add this feature, is to add a separate DB table for Social Profile Links, which contains Url, Id and User_id columns. Then Ruby on Rails will do parsing to distinguish different kinds of websites and links. And finally, front-end will use Bootstrap Icons to show visual part. There won't be needed any additional gems or libraries.

Visual Idea:
I think, it will be a smoother UX if user can change Social Profile Links while directly standing on the profile page (without redirecting on the separate page). It will add about 5 lines of JS, but I think, it's worth it.

I'll open a PR after the implementation will be in accordance with the ideas discussed in this issue.

Example screenshots:

![image](https://github.com/user-attachments/assets/8bd0752c-fe5a-4ae9-8cb2-fdd0220c35e4)

![image](https://github.com/user-attachments/assets/66a416f3-0d5a-4452-931a-3c653c1770cf)

**Comment by AntonKhorev** [comment]
The actual practical reason is to have `rel=me` automatically placed on these links.

**Comment by 1ec5** [comment]
This feature was implemented in #5439. We have some other tail work tickets about the feature, but this issue can be closed, right?

**Comment by tomhughes** [comment]
Looks like it, yes.
